articles/custom_storage_location.md

```diff
@@ -350,93 +351,10 @@ project_destination = syn.restPOST('/projectSettings', body = json.dumps(project
 
 ##### R
 
-Please see the [REST docs](http://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/project/ExternalS3StorageLocationSetting.html) for more information on setting external storage location settings using our REST API.
```

Comment:
This content was duplicated so it appeared twice in this document, hence the deletion